File: src/main/kotlin/com/cengenes/kotlin/api/controller/IndexController.java
package com.cengenes.kotlin.api.controller;

import org.springframework.stereotype.Controller;
import org.springframework.web.bind.annotation.GetMapping;

@Controller
public class IndexController {

    @GetMapping("/")
    public String redirectToSwaggerUi(){
        return "redirect:/swagger-ui.html";
    }

}

File: src/main/kotlin/com/cengenes/kotlin/api/controller/IndexController.kt
package com.cengenes.kotlin.api.controller

import org.springframework.stereotype.Controller
import org.springframework.web.bind.annotation.GetMapping

@Controller
class IndexController {

    @GetMapping("/")
    fun redirectToSwaggerUi(): String {
        return "redirect:/swagger-ui.html"
    }
}
